images/github-cli/Dockerfile

```diff
@@ -7,7 +7,17 @@ RUN apt-get update -qq && apt-get install -y ca-certificates git wget
 ADD https://cli.github.com/packages/githubcli-archive-keyring.gpg /usr/share/keyrings/githubcli-archive-keyring.gpg
 RUN chmod 644 /usr/share/keyrings/githubcli-archive-keyring.gpg
 RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" > /etc/apt/sources.list.d/github-cli.list
-RUN apt-get update -qq && apt-get install -y gh
+RUN echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_11/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
+RUN apt-get update -qq && \
+    apt-get install -y \
+    gh \
+    awscli \
+    curl wget gnupg2 -y \
```

Comment:
nit: duplicate `-y`

Reply:
Ahh thanks, didn't think the first -y flag would apply to the wget command.
I'll fix that.